validator/api/src/main/java/tech/pegasys/teku/validator/api/required/SyncingStatus.java

```diff
@@ -59,6 +59,10 @@ public Optional<Boolean> isElOffline() {
     return elOffline;
   }
 
+  public boolean isReady() {
+    return !elOffline.orElse(false) && (!isSyncing() || isOptimistic.orElse(false));
```

Comment:
what about make this explicit (aka. more readable?)
like:

if(elOffline.isPresent() && elOffline.get()) {
 return false;
}
and so on.

@rolfyone is it safe to add additional methods here or there might be strange interaction with our serialization\deserialization framework. I think its safe since we explicitly reference builder\class methods in the definitions.

Reply:
Made it more readable.
I don't think it will influence the serialisation/de since we call the getters/setters and use the builder in the type definitions

Reply:
Yet more readable...